aixtend/modules/metric.py

```diff
@@ -0,0 +1,91 @@
+__author__='shreyassharma'
```

Comment:
File naming can be consistent: datasets.py, metric -> metrics.py, model -> models.py, and so forth.